modules/402-ingress-nginx/hooks/minimal_ingress_nginx_version.go

```diff
@@ -88,6 +89,12 @@ func discoverMinimalNginxVersion(input *go_hook.HookInput) error {
 				isDisruptionUpdate = true
 			}
 		}
+		if ctrl.Version == deprecatedControllerVersion {
```

Comment:
it makes sense to add a couple of tests to check that relevant metrics get successfully created/deleted.

Reply:
and, as I remember, we already have a metric with controller version 
![image](https://github.com/deckhouse/deckhouse/assets/41360396/efd1195d-9e44-43a2-bfad-d5b73204e03c). Could it be better just to add an alert checking if `controller_version` labels equals `1.1`?